client/my-sites/plans-features-main/index.jsx

```diff
@@ -434,6 +439,41 @@ export class PlansFeaturesMain extends Component {
 		return false;
 	}
 
+	getProductsSelectorSubHeader() {
+		const { purchases, siteSlug, translate } = this.props;
+		const baseCopy = translate( "Just looking for a backups? We've got you covered." );
+		const jetpackBackup =
+			purchases && purchases.find( purchase => purchase.active && isJetpackBackup( purchase ) );
+
+		// Don't attach a link to a landing page if a user already has a Jetpack Backup product.
+		if ( jetpackBackup ) {
+			return baseCopy;
+		}
+
+		// If we have a site in this context, add it to the landing page URL.
+		const linkUrl = siteSlug
+			? addQueryArgs( { site: siteSlug }, JETPACK_BACKUP_PRODUCT_LANDING_PAGE_URL )
+			: JETPACK_BACKUP_PRODUCT_LANDING_PAGE_URL;
+
+		return (
+			<Fragment>
+				{ baseCopy }
+				<br />
```

Comment:
Since this is part of a paragraph tag, a simple `<br />` is the most suitable solution here I guess.